feat/3: Added new config to allow users to toggle total card counters

jbarbosa7 opened the pull request:
This PR fixes #3 

- Added a new config to allow users to toggle the total card counters labels

Balastrong reviewed (changes requested):
Thank you for the PR, I see the code style of the project is perfectly matched, well done!

I added a little note in the other comment, everything else looks good :)

jbarbosa7 commented on public/popup.html:
```diff
@@ -13,6 +13,15 @@
       <div class="config-section">
         <!-- Reenable if headers will make sense -->
         <!-- <div class="config-section-title">Card Numbers</div> -->
+        <div class="config-row">
+          <div class="config-label">Card Counters</div>
+          <div class="config-control">
+            <label class="switch">
+              <input type="checkbox" id="card-counters-active" />
+              <span class="slider round"></span>
+            </label>
+          </div>
+        </div>
         <div class="config-row">
           <div class="config-label">Active</div>
```
Great catch! I actually thought about that but then forgot to change it 😓 I'll update this as soon as i can :)

jbarbosa7 commented:
> Thank you for the PR, I see the code style of the project is perfectly matched, well done!
> 
> I added a little note in the other comment, everything else looks good :)

@Balastrong 

> Thank you for the PR, I see the code style of the project is perfectly matched, well done!
> 
> I added a little note in the other comment, everything else looks good :)

Already updated the PR with your sugestion :)

Balastrong reviewed (approved):
Thank you :D
